### DISCUSSION TITLE
Layer per image ?

### DISCUSSION THREAD

**Discussion by MentalGear** [konvajs/konva#1504] [Q&A] [answered]
First, thank you for this awesome and super helpful canvas library!

### Question:  re: Performance
Is it good practice to add for each (user uploaded) image an own layer, or put them all in one layer?
Individual images should be allowed to be alpha adjusted .

**Comment by lavrton** [comment, marked as answer]
Usually, not a good idea. Better to use as fewer layers as possible. 1-3 is ideal.

You may do photoshop style layer list with the plan images too and control their z index.

**Reply by MentalGear** [reply]
Makes sense, thx for the great tips !
